pkgs/os-specific/linux/ax99100/default.nix

```diff
@@ -0,0 +1,28 @@
+{ kernel, stdenv, kmod, lib }:
+stdenv.mkDerivation
+{
+  pname = "ax99100";
+  version = "1.8.0";
+  nativeBuildInputs = [ kmod ] ++ kernel.moduleBuildDependencies;
+  src = builtins.fetchTarball {
```

Comment:
use pkgs.fetchzip here instead. builtins.fetch* is disallowed on hydra/ofborg because it's IFD

Reply:
Ah, nice!

Thanks for the quick answer.